distribution/docs/src/main/resources/content/_installing/installing-intro.adoc

```diff
@@ -38,8 +38,8 @@ These are the system/environment requirements to configure _prior_ to an install
 |*Recommended*
 
 |Operating System
-|Windows Server 2012, Linux CentOS 5, Mac OS X 10.9 Mavericks
-|Windows 10, Linux CentOS 7.3. macOS Sierra 10.12
+|Windows 10, Linux CentOS 5
```

Comment:
We might want to require CentOS 6+ since CentoOS 5 hit [EOL](https://lists.centos.org/pipermail/centos-announce/2017-April/022350.html) on March 31

Reply:
Also our minimum of Windows 10 is technically equivalent to Windows Server 2016, but then our recommended section states Windows Server 2012 which is equivalent to Windows 8 ( or 8.1 if we are talking Server 2012R2)